src/pages/Index/index.spec.tsx

```diff
@@ -8,6 +8,6 @@ describe(__filename, () => {
   it('renders a page', () => {
     const root = shallow(<Index />);
 
-    expect(root.find(Link)).toHaveLength(1);
+    expect(root.find(Link)).toHaveLength(2);
```

Comment:
This test also doesn't seem helpful. I would say `expect(root.find(Link)).toExist()` is good enough.